Directory.Build.targets

```diff
@@ -34,68 +34,18 @@
     </Otherwise>
   </Choose>
 
-  <PropertyGroup>
-    <!-- When building from a dev machine, the BuildNumber won't be set, so set it to 0.1 to indicate this is a private build -->
-    <BuildVersion>0</BuildVersion>
-
-    <!-- Get build number from TFS if in automated build -->
-    <!-- Use TF_BUILD_BUILDNUMBER for old XAML style builds -->
-    <BuildVersion Condition=" '$(BUILD_BUILDNUMBER)' != '' ">$(BUILD_BUILDNUMBER.Split('.')[1].TrimStart('0'))</BuildVersion>
-    <BuildVersion>$(BuildVersion.PadLeft(5,"0"))</BuildVersion>
-
-    <!-- Get version info from file -->
-    <VersionFile>$(MSBuildProjectDirectory)\Properties\version.txt</VersionFile>
-    <Version>1.0.0</Version>
-    <Version Condition="Exists('$(VersionFile)')">$([System.IO.File]::ReadAllText($(VersionFile)).Trim())</Version>
-
-    <!-- Set informational version -->
-    <InformationVersion>$(Version)</InformationVersion>
-    <InformationVersion Condition=" '$(VersionSuffix)' != '' ">$(InformationVersion)-$(VersionSuffix)-$(BuildVersion)</InformationVersion>
-    <BasicVersion>$(Version)</BasicVersion>
-    <Version Condition=" '$(VersionSuffix)' != '' ">$(Version)-$(VersionSuffix)</Version>
-  </PropertyGroup>
-
-  <ItemGroup>
-    <AssemblyVersionAttribute Include="System.Reflection.AssemblyVersionAttribute">
-      <_Parameter1>$(BasicVersion)</_Parameter1>
-    </AssemblyVersionAttribute>
-    <AssemblyVersionAttribute Include="System.Reflection.AssemblyFileVersionAttribute">
-      <_Parameter1>$(BasicVersion)</_Parameter1>
-    </AssemblyVersionAttribute>
-    <AssemblyVersionAttribute Include="System.Reflection.AssemblyInformationalVersionAttribute">
-      <_Parameter1>$(InformationVersion)</_Parameter1>
-    </AssemblyVersionAttribute>
-    <AssemblyVersionAttribute Include="System.Reflection.AssemblyTitle">
-      <_Parameter1>$(AssemblyName)</_Parameter1>
-    </AssemblyVersionAttribute>
-    <AssemblyVersionAttribute Include="System.Reflection.AssemblyProduct">
-      <_Parameter1>$(AssemblyName)</_Parameter1>
-    </AssemblyVersionAttribute>
-    <AssemblyVersionAttribute Include="System.Reflection.AssemblyCopyright">
-      <_Parameter1>Copyright 2016</_Parameter1>
-    </AssemblyVersionAttribute>
-    <AssemblyVersionAttribute Include="System.Reflection.AssemblyCompany">
-      <_Parameter1>Microsoft</_Parameter1>
-    </AssemblyVersionAttribute>
-  </ItemGroup>
-
   <ItemGroup>
     <PackageReference Include="MicroBuild.Core" Version="0.2.0" Condition="!$(IsTest)">
       <PrivateAssets>All</PrivateAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.3.0-beta1">
       <PrivateAssets>All</PrivateAssets>
     </PackageReference>
+    <PackageReference Include="GitVersionTask" Version="4.0.0-beta0012" Condition=" '$(SkipVersion)' != 'true' ">
```

Comment:
Instead of using `SkipVersion` variable, maybe we can keep track of which MSBuild we are using and choose whether or not to import the NuGet files based on that. maybe it'll be better to add something like this:

```
  <!-- GitVersionTask is not compiled against .NET Core,
    importing the targets will result in a failure because it will try to
    resolve Microsoft.Build.Utilities v4.0 -->
  <PropertyGroup Condition="'$(MSBuildRuntimeType)' == 'Core'">
    <ExcludeRestorePackageImports>true</ExcludeRestorePackageImports>
  </PropertyGroup>
```

Reply:
Cool - I like it

Reply:
What is the ExcludeRestorePackageImports? I ended up just adding it as a condition on the PackageReference. Thoughts?

Reply:
@twsouthwick When you install package references, sometimes it comes with `<PackageName>.nuget.targets` or `<PackageName>.nuget.props`. This MSBuild property excludes those from being imported during build time because usually they're imported as msbuild runs through the csproj.

Reply:
Ah - so wouldn't the way you recommend end up breaking some that do support .net core?

Reply:
Good point. Looking for msbuildRuntimetype would be better!